src/lib/components/workflow/workflow-counts.svelte

```diff
@@ -116,24 +123,43 @@
     }
   };
 
+  const onStatusClick = (status) => {
+    const filter = {
+      attribute: 'ExecutionStatus',
+      type: SEARCH_ATTRIBUTE_TYPE.KEYWORD,
+      value: status,
+      operator: '',
+      conditional: '=',
+      parenthesis: '',
+    };
```

Comment:
Do we have a type or a function to generate this filter?

Reply:
It's all tangled up in the Context for the workflow filter search stuff. But it's typed and would blow up if in the $workflowFilters = [...$workflowFilters, filter] logic